model/model.py

```diff
@@ -35,34 +36,32 @@ def get_current_floor(self) -> int:
         pass
 
 
-
 class Bellhop(BellhopModelInterface):
 
-    def __init__(self, num_floors: int, capacity: int):
+    def __init__(self, level: Level):
+        self.level = level
         self._curr_state: State = State.WAIT_INPUT
         self._next_state: State = State.WAIT_INPUT
         self._state_leave: float = time.time() + float('inf')
         self._user_input: Direction = None
+        self._move_count = 0
 
         # floors
-        self._num_floors: int = num_floors
+        self._num_floors: int = self.level.get_num_floors()
         self._curr_floor: int = 0
 
-
         # people
         self._passengers: GaggleOfPassengers = GaggleOfPassengers()
-        # todo enforce capacity
-        self._capacity: int = capacity
+        self._capacity: int = self.level.get_capacity()
 
 
     def step(self, user_input: Optional[Direction]):
         self._user_input = user_input
 
-        self.make_random_passenger()
         if self._curr_state == State.ARRIVING:
             self._setup_next_state(State.PEOPLE_OFF, STATE_TIME_ARRIVAL_SECONDS)
             if self._state_timeout():
-                # more actions?
+                self.make_passengers_from_schedule()
```

Comment:
I know we discussed this when you were adding it, but from a user point of view wouldn't this potentially show up as unexpected passengers on the floor they just got to? ex. Arrive at 2, expecing ppl going up. Get there, they go both directions - when do you decide whether to let the down passengers in? I guess that's a totally reasonable mechanic, probably better to revisit when we can actually play the game just noticed the placement here.